charts/pulsar/templates/oxia-server-podmonitor.yaml

```diff
@@ -19,5 +19,5 @@
 
 # deploy oxia-server PodMonitor only when `$.Values.oxia.server.podMonitor.enabled` is true
 {{- if and $.Values.components.oxia $.Values.oxia.server.podMonitor.enabled }}
-{{- include "pulsar.podMonitor" (list . "oxia.server" "app.kubernetes.io/component: oxia-server" "metrics") }}
```

Comment:
one more detail is that "metrics" should be passed as the 4th parameter. That's the port name to be used.

Reply:
Thanks. It's added now.